rcl/test/CMakeLists.txt

```diff
@@ -0,0 +1,94 @@
+if(AMENT_ENABLE_TESTING)
```

Comment:
I was concerned about how `add_subdirectory` behaves if I rerun cmake without `AMENT_ENABLE_TESTING` set, after having run it with it set. I'll remove it. b92148d